web/src/components/apps/AppVersionHistory.jsx

```diff
@@ -19,10 +19,17 @@ import AppVersionHistoryRow from "@src/components/apps/AppVersionHistoryRow";
 import DeployWarningModal from "../shared/modals/DeployWarningModal";
```

Comment:
commented lines are the only ones I've changed in this PR. 
everything else is by the linter. :(